pythermalcomfort/environment/ground_temperature.py

```diff
@@ -0,0 +1,5 @@
+"""
+This module handles helper methods to estimate ground temperatures
```

Comment:
please do not include empty files for now, we can add them later when needed, this speeds up the review process